app/controllers/schools_controller.rb

```diff
@@ -129,4 +132,9 @@ def students_for_dashboard(school)
     school.students.active
   end
 
+  def school_year_start
+    current_year = Date.today.year
+    Date.today.to_s > current_year.to_s + "-08-01" ? current_year.to_s + "-08-01" : (current_year -1).to_s + "-08-01"
```

Comment:
Looks like we're comparing strings to strings here. This makes me pause because comparing numbers (or date times) as strings can work unpredictably: 

```
2.3.0 :001 > '11111' > '11'
 => true 
2.3.0 :002 > '11111' > '22'
 => false 
```

If we work with dates here, I think we should compare them as dates. We could also just remove the concept of student school year from the server side altogether and just send the last 1 year or 2 years of data to the client. Then the client can slice things up into school years if it wants to.

Reply:
It's moot now that the school year logic is moved out of here, but I wanted to point out that while I do agree it's better generally to compare dates as dates, as long as they're formatted in descending order (eg. YYYY-MM-dd), it should be alright. In this case it seemed faster to construct the strings. But you're right, it's safer to just compare dates.